client/src/components/Shop/shop.js

```diff
@@ -0,0 +1,270 @@
+import React, { Component } from "react";
+// TODO: Create products db to hold prod data
+// FIXME: import API from "../../utils/API";
+import { Link } from "react-router-dom";
+import Button from "react-bootstrap/Button";
+import MyVerticallyCenteredModal from "../Modals/index";
+
+let productList = [
+  { name: "Ice Cream", price: 1, info: "Let's get some ice cream!" },
+  {
+    name: "Popcorn & Movie",
+    price: 10,
+    info: "Ready to watch a movie? Popcorn included!"
+  }
+];
+
+/* Product */
```

Comment:
oh boy 3 classes in 1 file this time. plz no more. these are each a separate component take them out to their own files and import them where necessary.

Reply:
The shop.js component is being split to multiple components. (Work in Progress) - this was suppose to be a placeholder